### PR TITLE
Use chart.js for job statistics

### PR DESCRIPTION
**Pull request by vuminhhieu1311**
## Related Tickets
- [#41521](https://edu-redmine.sun-asterisk.vn/issues/41521)
## WHAT (optional)
- Use char.js for statistic
## Evidence (Screenshot or Video)
Screen Shot 2021-09-09 at 21 38 58
Screen Shot 2021-09-09 at 21 03 31
Screen Shot 2021-09-09 at 21 02 57

### CODE REVIEW

File: app/Repositories/Job/JobRepository.php [hunk 1]
```diff
@@ -71,4 +72,62 @@ public function searchByKeyword($keyword)
...
+        }
+
+        return $months;
+    }
+
+    public function getMonthlyJobCount($month)
+    {
+        return Job::whereYear('created_at', date('Y'))
+            ->whereMonth('created_at', $month)->get()->count() * 10;
+    }
```
Comment (nvvu99) [inline app/Repositories/Job/JobRepository.php:98]: Số 10 nghĩa là gì vậy?

File: app/Repositories/Job/JobRepository.php [hunk 2]
```diff
@@ -71,4 +72,62 @@ public function searchByKeyword($keyword)
...
+
+    public function getJobTypeCount($type, $status)
+    {
+        $attributes = [
+            'job_type' => $type,
+            'status' => $status,
+        ];
+
+        return Job::where($attributes)->count() * 10;
+    }
```
Comment (nvvu99) [inline app/Repositories/Job/JobRepository.php:113]: Tương tự

File: resources/js/chart.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+		//Bar Chart
+		var barChart = document.getElementById("bar-chart");
+		var barCtx = barChart.getContext('2d');
+		barChart.height = 120;
+		var barConfig = new Chart(barCtx, {
+			type: 'bar',
+			data: {
+			labels: typeData.types,
+			datasets: [{
+				label: 'Active',
```
Comment (nvvu99) [inline resources/js/chart.js:15]: Indent 4 spaces

Comment (vuminhhieu1311) [reply resources/js/chart.js:15]: ok

File: app/Repositories/Job/JobRepository.php (hunk 2 above)
Comment (vuminhhieu1311) [reply app/Repositories/Job/JobRepository.php]: e quên ko xóa

File: app/Repositories/Job/JobRepository.php (hunk 1 above)
Comment (vuminhhieu1311) [reply app/Repositories/Job/JobRepository.php]: e quên ko xóa

**Comment by vuminhhieu1311** [comment]
ready

**Review by manhtd-1083 — APPROVED** [review]

**Pull request merged**